content/source/docs/enterprise/private/config.html.md

```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
 In all examples below, be sure to replace "`<TFE HOSTNAME>`" with the hostname
 of your Private Terraform Enterprise instance.
 
-Navigate to `https://<TFE HOSTNAME>/admin/bootstrap` in your browser. You will
-be presented with a status page that looks like this:
+Navigate to `https://<TFE HOSTNAME>:8800/` in your browser. You will
+be presented with the installer's dashboard that looks like this:
 
-![Bootstrap Status Page](./assets/bootstrap-status.png)
+![PTFE Installer Dashboard](./assets/post-install-repl-console.png)
 
 ### Creating an Administrator
 
-After clicking on "Begin Installation" you will be brought to a page called
-"Create Administrator". Enter the requested information to create the first
-Administrator account. You will be able to create additional administrators
-later.
+After clicking on the "Open", right below the "Stop Now" button, you will
+be brought to a page asking you to create the first PTFE administrator account.
```

Comment:
> After clicking "Open" below the button labeled "Stop Now", you'll see a page where you can create the first PTFE admin user.

Reply:
As a side note - do we have any control over what the button says? "Open" is pretty vague.